app/controllers/api/v1/games_controller.rb

```diff
@@ -1,16 +1,26 @@
 module Api::V1
   class GamesController < ApplicationController
 
+    before_action :conflict_error, if: -> { lobby.game.present? },
+                                   only: [ :create ]
+
     def create
-      game = Game.create
-      game.add_user(current_user)
-      render json: game, serializer: GameSerializer
+      render json: lobby.create_game, serializer: GameSerializer
     end
 
     def show
-      game = Game.find(params[:id])
       render json: game, serializer: GameSerializer
     end
 
+    private
+
+    def game
+      @game ||= Game.find(params[:id])
```

Comment:
any authorization for viewing games? Should I be able to see game data for games that I have no part in?